src/wazuh_db/wdb_upgrade.c

```diff
@@ -122,6 +133,13 @@ wdb_t * wdb_upgrade_global(wdb_t *wdb) {
         }
     }
 
+    if (post_upgrade) {
+        if (OS_SUCCESS == wdb_global_create_backup(wdb, output)) {
+            mdebug1("Create post-upgrade global DB snapshot %s", output+3);
```

Comment:
We should change the `Create` for `Creating` or `Created`

Reply:
Thanks, changed!